Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/PDFPresentationController.mm

```diff
@@ -116,27 +116,39 @@
     ASSERT(pageContainerLayer);
     ASSERT(pageBackgroundLayer);
 
-    pageContainerLayer->setAnchorPoint({ });
-    addLayerShadow(*pageContainerLayer, containerShadowOffset, containerShadowColor, containerShadowStdDeviation);
+    if (shouldAddPageBackgroundLayerShadow()) {
+        pageContainerLayer->setAnchorPoint({ });
+        addLayerShadow(*pageContainerLayer, containerShadowOffset, containerShadowColor, containerShadowStdDeviation);
```

Comment:
We only want to limit these calls behind `shouldAddPageBackgroundLayerShadow()` right?

Reply:
...yes, we do. Oops!